arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -4534,7 +4558,7 @@
 					"@executable_path/Frameworks",
 				);
 				OTHER_LDFLAGS = "";
-				PRODUCT_BUNDLE_IDENTIFIER = "com.esri.arcgis-ios-sdk-samples";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.esri.arcgis-ios-sdk-samplesV";
```

Comment:
Is there a reason why the bundle identifier is changes?

Reply:
It was Divesh's solution that allowed me to deploy this app on a device. I'm currently resolving this situation with Nick. But I had reverted the bundle identifier in the general tab in my most recent commit, do I have to change the bundle identifier somewhere else in order to fix this?